src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -342,4 +342,16 @@
   <data name="PX1054Title" xml:space="preserve">
     <value>A PXGraph instance should not start PXLongOperation during the initialization phase</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
+    <value>Make the field bound to identity column the only key field in the DAC</value>
```

Comment:
@k-s-popova , please verify string

Reply:
@saloponov , consider to change as follows:
Use the field bound to the identity column as the only key field in the DAC